taskcat/_lambda_build.py

```diff
@@ -181,14 +182,23 @@ def _clean_build_log(line):
             line = line["aux"]
         return str(line).strip()
 
-    def _docker_build(self, path, tag):
-        _, logs = self._docker.images.build(path=str(path), tag=tag)
+    @staticmethod
+    def _docker_get_base_url():
+        docker_url = "https://127.0.0.1:2375/"
+        if os.getenv("DOCKER_HOST"):
+            docker_url = os.getenv("DOCKER_HOST")
+        return docker_url
+
+    @staticmethod
+    def _docker_build(path, tag):
+        cli = APIClient()
         build_logs = []
-        for line in logs:
-            line = self._clean_build_log(line)
-            if line:
-                build_logs.append(line)
-        LOG.debug("docker build logs: \n{}".format("\n".join(build_logs)))
+        for line in cli.build(path=str(path), tag=tag):
+            build_logs.append(line)
+        output = ""
+        for line in build_logs:
+            print(line.decode())
+        LOG.debug("docker build logs: \n{}".format("\n".join(output)))
```

Comment:
What's the expected output here? I perceive this as outputting null info.

Reply:
[DEBUG  ] : docker run logs: 

[INFO   ] : Packaging lambda source from /Users/gargana/workspace/scratch/aws-quickstart/quickstart-amazon-eks/functions/source/HelmReleaseResource using docker image taskcat-build-6b7a06b174c25d63b87e0636beb2bfab
{"stream":"Step 1/8 : FROM golang:1.16-alpine"}
{"stream":"\n"}

{"stream":" ---\u003e c4c01bca4bb0\n"}
{"stream":"Step 2/8 : RUN apk --no-cache add py3-pip make git zip"}
{"stream":"\n"}

{"stream":" ---\u003e Using cache\n"}
{"stream":" ---\u003e 7bb85c545596\n"}
{"stream":"Step 3/8 : RUN pip3 install cloudformation-cli-go-plugin"}
{"stream":"\n"}

{"stream":" ---\u003e Using cache\n"}
{"stream":" ---\u003e 8d14af227da1\n"}
{"stream":"Step 4/8 : COPY . /build"}
{"stream":"\n"}

{"stream":" ---\u003e Using cache\n"}
{"stream":" ---\u003e a46d943f73fa\n"}
{"stream":"Step 5/8 : WORKDIR /build"}
{"stream":"\n"}

{"stream":" ---\u003e Using cache\n"}
{"stream":" ---\u003e c6de099fc892\n"}
{"stream":"Step 6/8 : RUN GOPROXY=direct go mod download"}
{"stream":"\n"}

{"stream":" ---\u003e Using cache\n"}
{"stream":" ---\u003e f450615c3dd6\n"}
{"stream":"Step 7/8 : RUN GOPROXY=direct make -f Makefile.package package"}
{"stream":"\n"}

{"stream":" ---\u003e Using cache\n"}
{"stream":" ---\u003e 04b42f72a2d7\n"}
{"stream":"Step 8/8 : CMD mkdir -p /output/ \u0026\u0026 mv /build/awsqs-kubernetes-helm.zip /output/"}
{"stream":"\n"}

{"stream":" ---\u003e Using cache\n"}
{"stream":" ---\u003e 3493bb2bdbcd\n"}
{"aux":{"ID":"sha256:3493bb2bdbcd38f76e3d5aea80fca29ece8f2b87be57e2ef0af8531945e13f94"}}

{"stream":"Successfully built 3493bb2bdbcd\n"}

{"stream":"Successfully tagged taskcat-build-6b7a06b174c25d63b87e0636beb2bfab:latest\n"}